cmd/hack.go

```diff
@@ -0,0 +1,68 @@
+package cmd
+
+import (
+  "log"
+
+  "github.com/Originate/gt/cmd/config"
+  "github.com/Originate/gt/cmd/git"
+  "github.com/Originate/gt/cmd/script"
+  "github.com/Originate/gt/cmd/steps"
+  "github.com/Originate/gt/cmd/util"
+
+  "github.com/spf13/cobra"
+)
+
+var abortFlag bool
+var continueFlag bool
+
+var hackCmd = &cobra.Command{
+  Use:   "hack",
+  Short: "Create a new feature branch off the main development branch",
+  Long:  `Create a new feature branch off the main development branch`,
+  Run: func(cmd *cobra.Command, args []string) {
+    steps.Run(steps.RunOptions{
+      Command: "hack",
+      IsAbort: abortFlag,
+      IsContinue: continueFlag,
+      IsSkip: false,
+      SkipMessage: "",
+      StepListGenerator: func() steps.StepList {
+        targetBranchName := checkPreconditions(args)
+        return getStepList(targetBranchName)
+      },
+    })
+  },
+}
+
+func checkPreconditions(args []string) string {
+  if len(args) == 0 {
+    util.ExitWithErrorMessage("No branch name provided.")
```

Comment:
The period and capitalization are based on the current implementation. I'd prefer to change the feature tests as little as possible at the moment. Also if we are changing that, I'd like to do that on the entire project. Can you please create a separate issue for this?